### DISCUSSION TITLE
How add Google / Facebook login with payloadCMS?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [payloadcms/payload#1969] [Q&A] [answered]
Hi, Can you you tell me an idea to add Google login button to `login page` of payload CMS?
I am planning let end user login to dashboard. 
Is there any way to do that?

**Comment by jacobsfletch** [comment, marked as answer]
Hey @IRediTOTO check out this comment: https://github.com/payloadcms/payload/discussions/1555#discussioncomment-4276115
